### PR TITLE
katacoda: Added demo for multitenancy.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

**Comment by yeya24** [comment]
Just played this demo and it is super cool!

File: tutorials/katacoda/thanos/7-multi-tenancy/step1.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: ['127.0.0.1:9091','127.0.0.1:9092']
+</pre>
+
+## Starting Fruit and Veggie Prometheus and Thanos Global View 
+
+Fruits and Veggies want to allow more Prometheus and replicas at some point so they want to have Thanos upfront, so let's quickly
```
Comment (bwplotka) [inline tutorials/katacoda/thanos/7-multi-tenancy/step1.md:54]: need to clean up this (:

File: tutorials/katacoda/thanos/7-multi-tenancy/step1.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+
+We call this "Tomato" problem a **Cross-tenant or Admin View** and it's a common case when teams/tenants are changing, users have access to multiple
+tenants data etc. This can be solved with another layer of global view: we know how to solve this problem from previous courses, we could
+start another Querier on top of our Team's Queries and open another endpoint just for Tomatoes. But here comes another problem...
+
+## Problem statement 2: Exclusive Infra for Each Tenant is very Expensive and does not scale.
+
+![diagram](./assets/no-read-tenancy-reuse.png)
+
+## Don't worry: Thanos was build with multi-tenancy in mind!
```
Comment (yeya24) [inline tutorials/katacoda/thanos/7-multi-tenancy/step1.md:228]: was `built`?

File: tutorials/katacoda/thanos/7-multi-tenancy/step2.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+![diagram](./assets/no-isolation.png)
+
+Both Prometeus and Thanos [follow UNIX philosopy](https://github.com/thanos-io/thanos#thanos-philosophy). **One of the principles is to ensure
+each component is doing one thing and do it well**. Thanos Querier does not perform any authentication or authorization.
+This is because you probably already have consistent auth mechanism in your organization. So why not composing that with flexible
+flat label pairs identifing the data blocks and each individual series for data isolation?
+
+### Meet [prom-label-proxy](https://github.com/prometheus-community/prom-label-proxy)
+
+At Red Hat we started [prom-label-proxy](https://github.com/prometheus-community/prom-label-proxy). This allows read tenancy for all the
```
Comment (yeya24) [inline tutorials/katacoda/thanos/7-multi-tenancy/step2.md:40]: Do we need to mention `Red Hat` here?

**Review by yeya24 — APPROVED** [review]

File: tutorials/katacoda/thanos/7-multi-tenancy/step2.md (hunk 3 above)
Comment (bwplotka) [reply tutorials/katacoda/thanos/7-multi-tenancy/step2.md:40]: Nope good point (:

**Pull request merged**